design/accepted/kv_migration.md

```diff
@@ -0,0 +1,45 @@
+# KV Migration (export import phase 1) Design
+
+## Definitions
+1. KV Migration - The transition from schema based postgres implementation to key value store based postgres implementation
```

Comment:
Does KV has to be over postgress implementation for that? Is there a difference in the definitions for other (future) "engines"?

Reply:
KV Migrate happens only on prior lakeFS installations using postgres and will use the same DB instance for migration.
After that - migrating from postgres KV store to another DB should be done via the export import feature